sdk/disconnectedoperations/Azure.ResourceManager.DisconnectedOperations/api/Azure.ResourceManager.DisconnectedOperations.net8.0.cs

```diff
@@ -0,0 +1,441 @@
+        public static Azure.ResourceManager.DisconnectedOperations.ImageData ImageData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, Azure.ResourceManager.DisconnectedOperations.Models.ImageProperties properties = null) { throw null; }
+        public static Azure.ResourceManager.DisconnectedOperations.Models.ImageDownloadResult ImageDownloadResult(Azure.ResourceManager.DisconnectedOperations.Models.ResourceProvisioningState? provisioningState = default(Azure.ResourceManager.DisconnectedOperations.Models.ResourceProvisioningState?), string releaseVersion = null, string releaseDisplayName = null, string releaseNotes = null, System.DateTimeOffset releaseOn = default(System.DateTimeOffset), Azure.ResourceManager.DisconnectedOperations.Models.ReleaseType releaseType = default(Azure.ResourceManager.DisconnectedOperations.Models.ReleaseType), System.Collections.Generic.IEnumerable<string> compatibleVersions = null, string transactionId = null, System.Uri downloadLink = null, System.DateTimeOffset linkExpiry = default(System.DateTimeOffset)) { throw null; }
+        public static Azure.ResourceManager.DisconnectedOperations.Models.ImageProperties ImageProperties(Azure.ResourceManager.DisconnectedOperations.Models.ResourceProvisioningState? provisioningState = default(Azure.ResourceManager.DisconnectedOperations.Models.ResourceProvisioningState?), string releaseVersion = null, string releaseDisplayName = null, string releaseNotes = null, System.DateTimeOffset releaseOn = default(System.DateTimeOffset), Azure.ResourceManager.DisconnectedOperations.Models.ReleaseType releaseType = default(Azure.ResourceManager.DisconnectedOperations.Models.ReleaseType), System.Collections.Generic.IEnumerable<string> compatibleVersions = null) { throw null; }
+    }
+    public partial class ArtifactDownloadResult : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.DisconnectedOperations.Models.ArtifactDownloadResult>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.DisconnectedOperations.Models.ArtifactDownloadResult>
```

Comment:
we should update the name here to be consistent with the name change of `ArtifactResource`,